### DISCUSSION TITLE
I try newArch and downloadBoost runs twice same time

### DISCUSSION THREAD

**Discussion by pnthach95** [software-mansion/react-native-reanimated#3546] [Q&A] [answered]
So RN 0.70 is just released and I try it with `newArchEnabled=true` and Reanimated 2.10.0. It runs downloadBoost twice same time, see screenshot below:

<img width="734" alt="Screen Shot 2022-09-06 at 9 50 37 AM" src="https://user-images.githubusercontent.com/31266357/188538361-f53ff1e4-ced9-450e-b747-d47756044f0d.png">

Is there a way to make it run once?

**Comment by tomekzaw** [comment, marked as answer]
Hey, you're right, we could use the boost archive downloaded by react-native. We plan to address this issue in near future.

As for now, you can set environmental variable `REACT_NATIVE_DOWNLOADS_DIR` and both React Native and Reanimated will check if boost, glog etc. is present there:
```sh
mkdir -p /Users/<username>/.react-native/downloads
```
You can add this line to your `~/.zshrc` (or other):
```sh
export REACT_NATIVE_DOWNLOADS_DIR=/Users/<username>/.react-native/downloads 
```
Don't forget to apply the changes in the current console:
```sh
source ~/.zshrc
```
> Note: For existing project this trick works after project clean.

Related to #3523.
